drivers/ieee802154/ieee802154_cc1200.c

```diff
@@ -37,7 +37,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include "ieee802154_cc1200.h"
 #include "ieee802154_cc1200_rf.h"
 
-#if defined(CONFIG_IEEE802154_CC1200_GPIO_SPI_CS)
+#if DT_INST_SPI_DEV_HAS_CS_GPIOS(0)
```

Comment:
I'm feeling really good about the decision to put the enabled nodes first in the INST order as an implementation detail now.

Maybe we should consider making this behavior formally documented and tested?

Reply:
I don't see any particular draw back to that.